src/components/AddRecipe.tsx

```diff
@@ -0,0 +1,73 @@
+import React, { FC, useState, ChangeEvent } from 'react';
+
+import {IRecipe} from '../models/IRecipe'
+
+export interface AddRecipeProps {
+  recipe: IRecipe;
+}
+
+export interface RecipeFormState {
+  title: string;
+  ingredients: string;
+  instructions: string;
+}
+
+type HTMLFormControls = HTMLInputElement | HTMLSelectElement | HTMLTextAreaElement;
+
+export const AddRecipe: FC<AddRecipeProps> = (props) => {
+
+  const [recipeForm, setRecipeForm ] = useState<RecipeFormState> ({
```

Comment:
Nit: inconsistent spacing inside the square brackets. Prettier should catch this...

Reply:
Hmm... at what point does prettier fix the changes? I manually resolved for now.